tests/components/test_nlp.py

```diff
@@ -426,6 +426,10 @@ def test_nbest_process_verbose(
 ]
 
 
+@pytest.mark.xfail(
+    strict=False,
+    reason="This test is flakey since sometime duckling does not recognize `sunday`",
```

Comment:
What do we know about when this happens? Can we alter the query or pass in a reference time to avoid the failure?